game.html

```diff
@@ -1,4 +1,3 @@
-
 <script type="text/javascript">
```

Comment:
Both files are '.html' extensions. If you'd like to keep them in two different files, one should be a '.js' extension and you should have a link to the JS file in your HTML.